internal/rootcoord/root_coord.go

```diff
@@ -1180,7 +1180,7 @@ func (c *Core) Start() error {
 		Params.CreatedTime = time.Now()
 		Params.UpdatedTime = time.Now()
 
-		c.stateCode.Store(internalpb.StateCode_Healthy)
+		c.UpdateStateCode(internalpb.StateCode_Healthy)
```

Comment:
We'd better add a UI to cover this case.

Reply:
Do you mean adding ut for `UpdateStateCode` function? Or add unit test to cover UpdateStateCode before performing any closing operations?